src/Pelagos/Bundle/AppBundle/Handler/UploadHandler.php

```diff
@@ -381,7 +381,7 @@ protected function removeDir($dir)
      */
     protected function toBytes($str)
     {
-        $val = trim($str);
+        $val = preg_replace('/\D/', '', trim($str));;
```

Comment:
Cool, I didn't know what \D did... But is is add \S also in brackets like [\S\D] you don't have to use the trim.

Reply:
Actually, the \S is not needed either. You do it all with the existing preg_replace, just remove the trim.